helm_deploy/hmpps-interventions-ui/templates/_envs.tpl

```diff
@@ -11,42 +11,42 @@ env:
     valueFrom:
       secretKeyRef:
         name: hmpps-auth
-        key: interventions-ui-client-id
+        key: interventions-ui-client-id.txt
```

Comment:
this is annoying - something must have gone wrong when i uploaded the secrets.

Reply:
I can also change it on the namespace side if you'd like?